assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -46,6 +71,20 @@ function TextPane(props) {
   }));
 
   const { showTextSets, showTextAndShapesSearchInput } = useFeatures();
+  const [textSets, setTextSets] = useState([]);
+
+  useEffect(() => {
+    if (showTextSets) {
+      loadTextSets().then((sets) => setTextSets(sets));
+    }
+  }, [showTextSets]);
+
+  const insertElements = useCallback(
+    (elements) => {
+      elements.forEach((e) => insertElement(e.type, e));
```

Comment:
This can create multiple history entries. I'd rather that we leave this alone for now and move it to #3966. Also, some text sets are top-aligned, others bottom-aligned.

Reply:
Note that I'm saying _"can"_, not _"will"_. Wrapping in `useBatchedCallback` would probably do the trick.